rotkehlchen/tests/api/test_exchanges.py

```diff
@@ -137,6 +147,37 @@ def test_setup_exchange(rotkehlchen_api_server):
     assert json_data['result'] == ['kraken', 'coinbasepro']
 
 
+@pytest.mark.parametrize('data, response_patch', [
+    # Bittrex
+    (
+        {'name': 'bittrex', 'api_key': 'ddddd', 'api_secret': 'fffff'},
+        UNSYNC_SYSTEM_CLOCK_RESPONSE_BITTREX,
+    ),
+])
+def test_setup_exchange_raises_unsync_system_clock_error(
+        rotkehlchen_api_server,
+        data,
+        response_patch,
+):
+    """Test that when setting up an exchange raises UnsyncSystemClockError the
+    response status code is 409 (from HTTPStatus.Conflict)
+
+    TODO update the `exchange` param items each time an existing exchange
```

Comment:
good point. As you said, make an issue for this, and point to this test in the issue to not forget the TODO